### DISCUSSION TITLE
Cyberduck redesign for Windows - Priority UI elements

### DISCUSSION THREAD

**Discussion by janwaelti98** [iterate-ch/cyberduck#14513] [Polls]
Dear Cyberduck Community,

We want to improve the user experience of Cyberduck on Windows 11 and update the user interface to meet Microsoft's current design guidelines. To ensure that we meet the needs and expectations of our community, we would like to involve you in the design process.

That's why we invite you to participate in our survey. We want to give you the opportunity to provide feedback on various UI elements to ensure Cyberduck is as user-friendly as possible. This first survey focuses on the basic layout of Cyberduck. We are also working on a new tab concept to better manage multiple connections. We have a few questions about this as well.

Your participation in this survey is essential for us to ensure that we can provide the best possible user experience. We would like to thank you in advance and look forward to your feedback.

Best regards,
The Cyberduck Team

<img width="709" alt="Screenshot 2023-04-11 at 16 07 46" src="https://user-images.githubusercontent.com/79445035/231189249-0bce79ea-6c34-408d-bf58-6a64950f01bb.png">

**Comment by dkocher** [comment]
An important aspect is that proficient users will almost always use keyboard shortcuts when possible. Thus these must be documented which is commonly only possible using a traditional menu bar.
